addons/rook/1.6.11/install.sh

```diff
@@ -213,7 +213,7 @@ function rook_cluster_deploy_upgrade() {
     # 1. https://rook.io/docs/rook/v1.6/ceph-upgrade.html#1-update-common-resources-and-crds
```

Comment:
All versions have exactly the same changes applied.
No code was change just how they are sorted out and the timeout.